src/components/common/ContactForm/view.jsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+
+import { Button, TextInput, DateInput, Dropdown } from '../';
```

Comment:
Common components should import component explicitly:
e.g. `import Button from '../Button';`